### ISSUE TITLE
Client crash

### ISSUE THREAD

**Issue by Aeron216** [LoliKingdom/LoliASM#76]
I really can't comprehend why I started getting this crash out of nowhere when reaching the main menu.

This were the only mods I updated:
![image](https://user-images.githubusercontent.com/54506307/165094454-384408c8-2192-42ca-8699-7b7fab444a59.png)

Some crashes and logs here:

1. 

[crash](https://gist.github.com/Aeron216/40297ebdc2cd807cba55cfe649f1456f)

[latest](https://gist.github.com/Aeron216/8f8160fa08e154ac821cee4f23c0a73b)

[debug](https://gist.github.com/Aeron216/e6e393fd52d378cb21f60a316e5eaac1)

Relaunched the game

2.

[crash](https://gist.github.com/Aeron216/c8dc71164b836c807732397e11e38601)

(Another crash within seconds)
[crash](https://gist.github.com/Aeron216/72dd36b93185e2865a05a4e9142f1c81)

[latest](https://gist.github.com/Aeron216/8713428b95c9356cafbb48ed7d650d1c)

[debug](https://gist.github.com/Aeron216/0021c94dfec7b063c3e0f6ae55746f7b)

Short clip:

https://gfycat.com/plainidolizedgrayling

Clean config file

I downgraded to the previous version I had before updating to 4.10.5 which is 4.10.3 and I kept getting the same crash, thats why I'm lost right now.

Edit:
I just downgraded the TrinkPlus mod and same thing. Damn.

However, I'm still able to enter any world and play normally.

[latest](https://gist.github.com/Aeron216/1479683a7fa84624e9c7840e470c075f)

**Comment by Aeron216** [comment]
Sorry man, getting a slightly different crash now. Still right after reaching the main menu.

[crash](https://gist.github.com/Aeron216/97e0e35de86e05cf8b49e626d52c0170)

[crash](https://gist.github.com/Aeron216/069e4e5c1433930eb6d337aa02646005)

[latest](https://gist.github.com/Aeron216/4116700cce71ea7d3450f40bba19f41a)

[debug](https://gist.github.com/Aeron216/2b7a1a61498feff629392d176a673ea8)

**Comment by Aeron216** [comment]
Huh, interesting

The other thing I did besides updating the aforementioned 3 mods, was to remove the Caliper mod.
https://www.curseforge.com/minecraft/mc-mods/caliper

Well, I just added it back and I'm not getting any client crashes any longer.
And I had been including that mod in my modpack for a very long time.

[latest](https://gist.github.com/Aeron216/e02cecfd18d842222463ec41a39f73ec)

[debug](https://gist.github.com/Aeron216/9f018f7452e4974295a025f9f2cbf8cd)
